.gitignore

```diff
@@ -3,7 +3,7 @@
 *.jl.*.cov
 *.jl.cov
 *.jl.mem
-Manifest.toml
+/Manifest.toml
```

Comment:
I think this need to be rebased.

Reply:
No, I need it to keep the Manifest of the benchmark comparison, otherwise it removes every Manifest file. I need the Manifest because I want reproducibility, like also the requirements.txt for the python packages.